lib/msf/core/handler/reverse.rb

```diff
@@ -82,12 +93,16 @@ def setup_handler
                 'MsfPayload' => self,
                 'MsfExploit' => assoc_exploit
               })
+
+          if is_loopback_address(ip)
```

Comment:
Please fix the indentation here. I had trouble reading the diff because it wasn't clear what went where.